builder/builder.go

```diff
@@ -1760,6 +1785,29 @@ func (b *Builder) BuildImage(format string, template string) error {
 	return imagecmd.Run()
 }
 
+// Updates bundle list in the default image template file
+func updateImageTemplate(filePath string, bundles []string) error {
+	f, err := os.Open(filePath)
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	byteValue, _ := ioutil.ReadAll(f)
```

Comment:
stop throwing away the errors in critical code paths.

Reply:
Fixed.